docker-java/src/test/java/com/github/dockerjava/cmd/StatsCmdIT.java

```diff
@@ -56,6 +57,40 @@ public void testStatsStreaming() throws InterruptedException, IOException {
 
     }
 
+    @Test
+    public void testStatsNoStreaming() throws InterruptedException, IOException {
+        TimeUnit.SECONDS.sleep(1);
```

Comment:
why is it here?

Reply:
Same as in testStatsStreaming() method. Maybe already not needed in both places.

Reply:
yeah, I think so. Especially at the beginning of the test where everything should already be prepared :)